src/test/scala/net/snowflake/spark/snowflake/MiscSuite01.scala

```diff
@@ -222,9 +222,9 @@ class MiscSuite01 extends FunSuite with Matchers {
     assert(sparkConfNode.get("spark.driver.memory").asText().equals("2G"))
     assert(sparkConfNode.get("spark.executor.memory").asText().equals("888M"))
     assert(sparkConfNode.get("spark.driver.extraJavaOptions")
-      .asText().equals("-Duser.timezone=GMT"))
+      .asText().contains("-Duser.timezone=GMT"))
```

Comment:
Spark 3.3 added more JVM options, so update the unit test o check `contains` instead of `equal`.